doc-content/enterprise-only/vehicle-route-planning/vrp-run-openshift.adoc

```diff
@@ -0,0 +1,52 @@
+[id='vrp-run-openshift_{context}']
+
+= Run OptaWeb Vehicle Routing on {OPENSHIFT} through  Red Hat CodeReady Containers
+
+Linux and macOS users can use the `runOnOpenShift.sh` Bash script to install OptaWeb Vehicle Routing on {OPENSHIFT}. You can use Red Hat CodeReady Containers to set up a single-node OpenShift 4 cluster on your local computer. CodeReady Containers provides a minimal preconfigured OpenShift cluster for development and testing purposes. It is delivered as a Red Hat Enterprise Linux virtual machine.
+
+.Prerequisites
+* OptaWeb Vehicle Routing has been successfully built with Maven as described in xref:download-ref-imp-proc_{context}[].
+* Internet access is available.
+
+
+.Procedure
+. To install and start Red Hat CodeReady Containers, follow the instructions in the link:https://code-ready.github.io/crc/[_Red Hat CodeReady Containers Getting Started Guide_].
+
+. When the OpenShift cluster starts, perform the following steps:
+
+.. Add the OpenShift command-line interface (`oc`) to your `$PATH`:
++
+[source,shell]
+----
+eval $(crc oc-env)
+----
+
+.. Log in as the `developer` user:
++
+[source,shell]
+----
+oc login -u developer -p developer https://api.crc.testing:6443
+----
+
+.. Enter the following command to create a new project where `<PROJECT_NAME>` is the name of your new project:
++
+[source]
+----
+oc new-project <PROJECT_NAME>
+----
+.. If necessary, change directory to the `{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-optaweb-vehicle-routing/sources` directory.
+
+.. Enter the following command to execute the `runOnOpenShift.sh` script:
++
+[source]
+----
+./runOnOpenShift.sh <OSM_FILE_NAME> <COUNTRY_CODE_LIST> <OSM_FILE_DOWNLOAD_URL>
+----
++
+In this command, replace the following variables:
+
+* `<OSM_FILE_NAME>`: The name of OSM file
```

Comment:
Maybe we could add a reference what OSM is? Currently it looks a bit self-repeating

Reply:
Added.